Sources/MongoSwift/Operations/Operation.swift

```diff
@@ -0,0 +1,8 @@
+/// A protocol for operation types to conform to. An `Operation` instance corresponds to any single operation a user
```

Comment:
I don't remember what the rules are for comments, but I think y'all moved to c-style comments at a certain point?

Reply:
we standardized on (from development guide):
> We use C-style documentation blocks (`/** ... */`) for documentation longer than 3 lines, and triple-slash (`///`) for shorter documentation.

Reply:
I've been doing single and double line docstrings as `///` and c-style for everything longer.